master/getting-started/kubernetes/upgrade/delete.md

```diff
@@ -3,5 +3,68 @@ title: Deleting old data
 no_canonical: true
 ---
 
-_Docs on this coming soon!_
+## About deleting old Calico data
 
+You may need or wish to manually delete Calico data from your etcd datastore under the
+following conditions.
+
+- [A data migration attempt failed partway through, leaving the etcdv3 datastore
+  with some, but not all of your etcvd2 data](#deleting-calico-data-from-etcdv3-after-a-partial-migration).
+  
+- [You succeeded in migrating your data and upgrading to Calico v3.0. After
+  running Calico for some time and experiencing no errors, you want to delete
+  the Calico data from the etcdv2 datastore](#deleting-calico-data-from-etcdv2-after-a-successful-migration-and-upgrade).
+  
+
+## Deleting Calico data from etcdv3 after a partial migration
+
+1. Obtain the name of the etcd pod. 
```

Comment:
This step is specific to a kubeadm install. 

etcd isn't always run as a pod (I'd recon the majority of cases outside of kubeadm do not run etcd as pod).

Reply:
Also, I don't think this step is even required for kubeadm - you can run etcdctl outside of the pod. Just need to direct users to the etcdctl install page first.